src/meta/src/barrier/mod.rs

```diff
@@ -785,55 +839,54 @@ where
         &self,
         node: &mut EpochNode<S>,
         tracker: &mut CreateMviewProgressTracker,
+        checkpoint_control: &mut CheckpointControl<S>,
     ) -> MetaResult<()> {
         let prev_epoch = node.command_ctx.prev_epoch.0;
-
         match &node.state {
             Completed(Ok(resps)) => {
                 // We must ensure all epochs are committed in ascending order,
                 // because the storage engine will query from new to old in the order in which
                 // the L0 layer files are generated.
                 // See https://github.com/singularity-data/risingwave/issues/1251
-                let mut sst_to_worker: HashMap<HummockSstableId, WorkerId> = HashMap::new();
-                let mut synced_ssts: Vec<LocalSstableInfo> = vec![];
-                for resp in resps {
-                    let mut t: Vec<LocalSstableInfo> = resp
-                        .synced_sstables
-                        .iter()
-                        .cloned()
-                        .map(|grouped| {
-                            let sst = grouped.sst.expect("field not None");
-                            sst_to_worker.insert(sst.id, resp.worker_id);
-                            (grouped.compaction_group_id, sst)
-                        })
-                        .collect_vec();
-                    synced_ssts.append(&mut t);
-                }
-
-                if prev_epoch == INVALID_EPOCH {
-                    assert!(
-                        synced_ssts.is_empty(),
-                        "no sstables should be produced in the first epoch"
-                    );
-                } else {
-                    self.hummock_manager
-                        .commit_epoch(prev_epoch, synced_ssts, sst_to_worker)
-                        .await?;
-                }
-
                 node.timer.take().unwrap().observe_duration();
```

Comment:
We will not take commit_epoch into consideration in the timer after this PR. Is this expected?

Reply:
Yes, because If it is a quick barrier, It will not commit_epoch and do post-processing work. Maybe we need to add new metrics to calculate this time period.

Reply:
Since we haven't introduced the not-checkpoint barrier in this PR, we should keep the timer metrics consistent with the previous one to avoid confusion.